keras_core/layers/preprocessing/normalization.py

```diff
@@ -160,6 +160,21 @@ def build(self, input_shape):
         mean_and_var_shape = tuple(input_shape[d] for d in self._keep_axis)
         self._mean_and_var_shape = mean_and_var_shape
 
+        self.mean = self.add_weight(
```

Comment:
This would break checkpoints. Is there another possible fix that would not break checkpoint structure?

Reply:
Changed to add a device check in `convert_to_tensor`.